Automattic/wp-calypso
# Refactor tabbed pricing page and implemens follow-up test layout

#### Changes proposed in this Pull Request
The changes in this PR are a refactor of the tabbed plans page first implemented in #56110. The refactor streamlines the CSS and assorted code shenanigans that came about during the iterative planning and development during the first test. 

This PR also implements the follow-up test described in pbxNRc-Zo-p2, namely:

* Adding the Premium plan to the Starter tab.
* Tweaking the feature selection on the Starter tab.

@southp this includes the work you saw in #56633, along with some additional improvements.

#### Testing instructions



* Checkout this PR and start Calypso. Alternatively, you can use the Calypso Live link below.
* Add yourself to the treatment variation of the `tabbed_layout_plans_signup_v2` Explat test (this will change when the new experiment is created).
* Navigate to /start/new and go through the signup flow.
* Test the flow using a variety of plan and term-length combinations.
* Complete the signup flow, including purchase, for both free and paid plans to confirm the flow still works as expected.
* Repeat the process at both viewports (above and below 660px).



Related to #56110
